File: python/Course.py
class Course():
  __namaMatkul = ''# atribut namaMatkul


  def __init__(self, namaMatkul = ''):
    # inisiasi dengan value awal
    self.__namaMatkul = namaMatkul
  
  def getNamaMatkul(self):
    # return namaMatkul
    return self.__namaMatkul
  
  def setNamaMatkul(self,namaMatkul):
    # mengganti value namaMatkul
    self.__namaMatkul = namaMatkul

File: c++/Course.cpp
class Course {
  private:
    string namaMatkul;// atribut namaMatkul

  public:
  Course(string namaMatkul = ""){
    // inisiasi dengan value awal
    this->namaMatkul = namaMatkul;
  }
  
  string getNamaMatkul(){
    // return namaMatkul
    return this->namaMatkul;
  }
  
  void setNamaMatkul(string namaMatkul){
    // mengganti value namaMatkul
    this->namaMatkul = namaMatkul;
  };
};